app/src/androidTest/java/com/epfl/beatlink/ui/profile/FakeSpotifyApiViewModel.kt

```diff
@@ -86,4 +86,8 @@ open class FakeSpotifyApiViewModel(
   ) {
     onResult("playlistId")
   }
+
+  override fun updatePlayer() {
+    // Simulate fetching playback state
+  }
```

Comment:
This addition doesn't seem relevant to this PR. I'm guessing that it's intended for a future task.

Reply:
it was needed because of the addition of the MusicPlayerUI in the SearchBarScreen, if not simulated, it would give us a NullPointerException when running tests for the SearchBarScreen